web/src/routes/admin/(resources)/students/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import DataTable, { type DataItem } from '$lib/elements/ui/dataTable/data-table.svelte';
+  import { deleteStudent, getStudents, type Pageable, type PageStudent } from '$lib/sdk/fetch-client';
+  import { error } from '@sveltejs/kit';
+  import { onMount } from 'svelte';
+  import { writable, type Writable } from 'svelte/store';
+
+  let columnNames = ['Vorname', 'Nachname', 'Tags'];
+  let keys = ['id', 'firstName', 'lastName', 'tags'];
+  let tableData: Writable<DataItem[]> = writable();
+  let totalElements: Writable<number> = writable(0);
```

Comment:
This whole store handling is honestly super weird. `loadPage` should just return the page and then you can do whatever with it.

Reply:
Found a way to avoid these stores, but could you take a look at the comment at the bottom of the students file where the data table component is. Maybe you have a better idea.